jte-spring-boot-starter-3/src/main/resources/META-INF/spring-configuration-metadata.json

```
@@ -30,6 +30,12 @@
       "type": "java.lang.String",
       "description": "You can configure the file suffix of jte templates the compiler resolves",
       "sourceType": "gg.jte.springframework.boot.autoconfigure.JteProperties"
+    },
+    {
+      "name": "gg.jte.exposeRequestAttributes",
```

just to be aligned with the recent change:

```suggestion
      "name": "gg.jte.expose-request-attributes",
```